x-pack/plugins/monitoring/public/views/cluster/overview/index.js

```diff
@@ -52,6 +53,8 @@ uiRoutes.when('/overview', {
         },
       });
 
+      this.init = () => this.renderReact(null);
```

Comment:
Again, maybe `ng-show` would be better

Reply:
I'm not sure `ng-show` will work as I'm waiting for the page to be ready before rendering the React component into the DOM